src/expandableCalendar/index.tsx

```diff
@@ -621,15 +621,4 @@ const ExpandableCalendar = (props: ExpandableCalendarProps) => {
 export default ExpandableCalendar;
 
 ExpandableCalendar.displayName = 'ExpandableCalendar';
-ExpandableCalendar.defaultProps = {
```

Comment:
should we fill in the rest of these as well in `src/calendar/header/index.tsx`? I think firstDay is the only one that's enumerated above so we maybe replace it with`firstDay = 0`.

Are the other props not used or why are they here?

Reply:
Oh looks like the other ones are ExpandableCalendar props. I think we're also missing `arrowsHitSlop`?